### DISCUSSION TITLE
Is there any way to remove GlobalScope in getTableFilters

### DISCUSSION THREAD

**Discussion by anilmbible** [filamentphp/filament#8507] [Help]
Hi Guys,

I have a global scope which I am applying to the model on boot function. Now I would like to perform withoutGlobalScope(SomeScope::class) in the getTableFilters. I would like to remove the scope if someone selects any of the filter

**Comment by leandrocfe** [comment]
You mean [disabling-global-scopes](https://filamentphp.com/docs/3.x/panels/resources/getting-started#disabling-global-scopes)?

**Reply by anilmbible** [reply]
Yes @leandrocfe

**Reply by leandrocfe** [reply]
Did you try it?
